.github/workflows/changelog.yml

```diff
@@ -33,7 +38,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v5
         with:
-          go-version: ~1.22.6
+          go-version: ${{ env.GO_VERSION }}
```

Comment:
those go versions could also be updated by renovate i guess. they don't seem to be picked up right now